README.md

```diff
@@ -128,6 +126,19 @@ To save DataFrame as avro you should use the `save` method in `AvroSaver`. For e
 ```scala
 scala> AvroSaver.save(myRDD, "my/output/dir")
 ```
+
+Avro schema's field 'doc' will be preserved in DataFrame's schema metadata.
+
+To include aliases column in scheme invoke
+```
+val df = sqlContext.read.format("com.databricks.spark.avro").option("withAlias", "true").load("test.avro")
+```
+In Spark SQL, loading avro with alias columns use:
+```
+create table mytable using com.databricks.spark.avro OPTIONS(path "test.avro", withAlias "true");
```

Comment:
make the reserved words `CREATE` `TABLE`, etc capital to match the rest of the documentation.